incubator/ethereum/templates/geth.configmap.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "ethereum.fullname" . }}-geth-config
+  labels:
+    app: {{ template "ethereum.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: geth-config
```

Comment:
It's not a component of its own. I'd remove the label.

Reply:
Fixed.